### DISCUSSION TITLE
addition of photos

### DISCUSSION THREAD

**Discussion by srinisri91** [naveenkumar-j/tomato-leaf-disease-prediction-ml-andriodapp#2] [Q&A]
could you please tell me that i would like to add the remedies but stucked with some errors...

**Comment by naveenkumar-j** [comment]
Hi @srinisri91, Once, you train your model to detect various diseases, you could alter this if else condition and you can add the remedies on the same block.
Please refer this video to understand how to build this app: [youtube link](https://www.youtube.com/watch?v=fqu1y7YvVWA)
![image](https://github.com/naveenkumar-j/tomato-leaf-disease-prediction-ml-andriodapp/assets/70682152/1f4c79c8-53e9-4f77-8787-3fe684c9af84)
